.env.template

```
@@ -4,8 +4,4 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_BOARDID=trello-id
-TRELLO_KEY=trello-key
-TRELLO_TOKEN=trello-token
-TRELLO_LISTID1=trello-listid1
-TRELLO_LISTID2=trello-listid2
+PRIMARY_CONNECTION=mongodb://module10:ulH6uBzQRqsLbVIWVR44Cil6nOwb6AML7ykXzCCBygQI4uvkn2Ok8yK1B3GjrhzCgjOE3LdGHJhkUObXOtpXaw==@module10.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@module10@
```

This is your real connection string, it shouldn't be in our repo!
If you visit your CosmosDB resource in Azure and click on the "Connection String" page you should see there's an option to regenerate your passwords so that this isn't valid going forward, but we should also empty this template - i.e. so it's just:
```suggestion
mongo_client=
```

Edit: Also, this isn't the environment variable name that your code actually looks for!

---

emptied